apps/cli/src/diagnostic/Timer.ts

```diff
@@ -26,11 +26,11 @@ export class Timer {
     return `${this._pad(hours)}:${this._pad(minutes)}:${this._pad(seconds)}`;
   }
 
-  toString() {
+  toString(): string {
     return this.getHHMMSS();
   }
 
-  toJSON() {
+  toJSON(): unknown {
```

Comment:
do we need this `toJSON` if we're returning a string? and if so shouldn't it be a stringified json (a la `JSON.stringify()`)?

Reply:
I think you might be confused a to what `toJSON` and `toString` do. We need `toJSON` defined here because we want the `Timer` struct to be represented as a "HH:MM:SS" string when it's serialized to JSON.

See the following code:
```typescript
const timer = new Timer()
const serialized = JSON.stringify(timer)

// "HH:MM:SS" string
console.log(serialized)
```